OpenOversight/app/templates/partials/officer_general_information.html

```diff
@@ -34,7 +34,7 @@ <h3>
           <td>
             <b>Department</b>
           </td>
-          <td>{{ officer.department.display_name }}</td>
+          <td>{{ officer.department.name }}</td>
```

Comment:
?

Reply:
Since we added grouping by state, it seemed cleaner to remove the display name